articles/tutorials/circuit-qft-tutorial.md

```diff
@@ -0,0 +1,298 @@
+﻿---
+title: Tutorial: Simulate a quantum circuit | Microsoft Docs
+description: Step-by-step tutorial to simulate a basic quantum circuit in Q#. Explore quantum gates and superposition.
+author: gillenhaalb
+ms.author: a-gibec@microsoft.com
+ms.topic: tutorial 
+ms.date: 10/06/2019
+uid: microsoft.quantum.circuit-tutorial
+# Customer intent: As a newcomer to the Q# (any amount of quantum experience), I want to know how to write and simulate a quantum circuit in Q# so that I can explore further on my own.
+---
+
+
+
+# Tutorial: Simulate quantum circuits in Q#
+Welcome to the Quantum Development Kit tutorial on setting up and simulating a basic quantum circuit. 
+It is important to note that the Q# language is far more than a new syntax for simulating quantum circuits. 
+If that were the only goal, a mere library of classical functions would have sufficed, because a quantum circuit is nothing more than the state's evolution through a specific sequence of gates---all of which can be fully expressed through matrix multiplication.
+Just as higher level classical languages like C and Python allow one to go beyond simulating sequences of AND and OR gates, Q# has been specifically designed with a computational paradigm shift in mind. 
+It is to be used for writing subroutines that, under the control of a classical host program and computer, execute on a separate quantum processing unit, thereby allowing developers to fully harness this classical-quantum symbiosis in full-scale quantum programs.
```

Comment:
I wouldn't think so, because most of this section was just freehand with some of Andres' intro Notebook brought in (e.g. line 19). Should it be?
Some of the info later on, like describing the C# driver, will be naturally very close to duplicate though